azurerm/resource_arm_batch_pool.go

```diff
@@ -232,6 +232,41 @@ func resourceArmBatchPool() *schema.Resource {
 								},
 							},
 						},
+
+						"resource_file": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"auto_storage_container_name": {
+										Type:     schema.TypeString,
+										Optional: true,
+										// TODO ValidateFunc - The autoStorageContainerName, storageContainerUrl and httpUrl properties are mutually exclusive and one of them must be specified.
```

Comment:
this'll need to be done during the Create/Expand function, unfortunately

Reply:
Ooops - I did the implementation in Create/Update but forgot to remove the comment!